eslint.config.mjs

```diff
@@ -0,0 +1,11 @@
+import js from "@eslint/js";
+import mocha from "eslint-plugin-mocha";
+export default [
+  mocha.configs.flat.recommended,
+  {
+    files: ["**/*.js"],
+    rules: js.configs.recommended.rules,
```

Comment:
just in case they ever specify more than rules, it may be worth using object spread:

```ts
{
  ...js.configs.recommended,
  files: ['**/*.js'],
}
```